x-pack/plugins/ingest_manager/common/services/is_agent_upgradeable.test.ts

```diff
@@ -135,4 +135,35 @@ describe('Ingest Manager - isAgentUpgradeable', () => {
       true
     );
   });
+  it('returns false if agent reports upgradeable, with agent snapshot version === kibana version', () => {
+    expect(
+      isAgentUpgradeable(getAgent({ version: '7.9.0-SNAPSHOT', upgradeable: true }), '7.9.0')
```

Comment:
are you also checking the option where agent reports
```
version: 7.9.0
upgradeable: true
snapshot: true
```

Reply:
Do you mean checking whether or not it is a snapshot?  It does not test for snapshot because it does not matter if its a snapshot whether or not its upgradeable (unless we decide to filter out snapshots).  But I can add the option anyway as part of the tests.  Let me know if I am misunderstanding...